scipy/sparse/_compressed.py

```diff
@@ -369,7 +386,13 @@ def __ge__(self, other):
 
     def _add_dense(self, other):
         if other.shape != self.shape:
-            raise ValueError(f'Incompatible shapes ({self.shape} and {other.shape})')
+            try:
+                # This will raise an error if the shapes are not broadcastable
+                bshape = np.broadcast_shapes(self.shape, other.shape)
+            except ValueError:
+                raise ValueError('inconsistent shapes')
```

Comment:
Can we just use the numpy error messages about broadcasting not being possible? Or are they not helpful in this situation. It would remove quite a few try/excepts leaving a 1 line instead of 5 in each case.

The message for `np.broadcast_shapes` looks like:
`ValueError: shape mismatch: objects cannot be broadcast to a single shape.  Mismatch is between arg 0 with shape (2, 3) and arg 1 with shape (5, 3).`

Reply:
Done at d05853538012f7ab99ebce49d6d51d77c273c821.